contracts/interfaces/IStakeManager.sol

```diff
@@ -19,11 +19,11 @@ interface IStakeManager {
         uint256 amount
     );
 
-    /// Emitted once a stake is scheduled for withdrawal
+    /// Emitted when stake or unstake delay are modified
     event StakeLocked(
         address indexed account,
         uint256 totalStaked,
-        uint256 withdrawTime
+        uint256 unstakeDelaySec
```

Comment:
Isn't it better to know the `withdrawTime` from the event? Maybe emit both values, actually?

Reply:
No, since when you lock, you don't really have withdrawTime, since it's only set upon unlocking the stake.